src/frontend/screens/Onboarding/walletSelection/screens/import/index.tsx

```diff
@@ -91,79 +99,90 @@ const ImportScreen = ({
                 `Access with secret recovery phrase`
               )}
               onClick={async () => {
-                handleImportMmExtensionClicked(null)
+                handleImportMmExtensionClicked('SECRET_PHRASE')
               }}
             />
           </NavLink>
         </div>
       ) : (
         <>
-          <div className={ImportScreenStyles.profileOptionsContainer}>
-            {Object.keys(importOptions[browserSelected]).map((pkgManager) => {
-              if (importOptions[browserSelected][pkgManager].length === 0)
-                return null
-              return (
-                <div
-                  className={ImportScreenStyles.packageOptionsContainer}
-                  key={`${pkgManager}-importOption-container`}
-                >
-                  <div
-                    className={`caption ${ImportScreenStyles.packageManagerSubTitle}`}
-                  >
-                    {pkgManager}
-                  </div>
-                  {importOptions[browserSelected][pkgManager].map(
-                    (profile: BrowserProfile, index: number) => (
-                      <button
-                        className={`${ImportScreenStyles.importBrowserProfileOption} body`}
-                        key={`${browserSelected}-${pkgManager}-${profile.name}-menu-item`}
-                        onClick={async () =>
-                          handleImportMmExtensionClicked(
-                            importOptions[browserSelected][pkgManager][index]
-                              .path
-                          )
-                        }
+          {
+            <div className={ImportScreenStyles.profileOptionsContainer}>
+              {Object.keys(importOptions[browserSelected]!).map(
+                (pkgManager) => {
+                  if (importOptions[browserSelected]![pkgManager].length === 0)
+                    return null
+                  return (
+                    <div
+                      className={ImportScreenStyles.packageOptionsContainer}
+                      key={`${pkgManager}-importOption-container`}
+                    >
+                      <div
+                        className={`caption ${ImportScreenStyles.packageManagerSubTitle}`}
                       >
-                        <div className={ImportScreenStyles.profileImage}>
-                          {profile.imagePath !== '' ? (
-                            <img src={`file://${profile.imagePath}`} />
-                          ) : (
+                        {pkgManager}
+                      </div>
+                      {importOptions[browserSelected]![pkgManager].map(
```

Comment:
and here

Reply:
also, we are trying to iterate an object which TS says might be undefined. If there is a change that it might come undefined, it might break the whole screen, no?

Reply:
the only way it sets the browser selected is from one of the importOptions keys or setting it as null. we never render this line if it is null. TypeScript just can't deduce this